### PR TITLE
Added Components to calculator

### PR DESCRIPTION
**Pull request by oscarnava**

### CODE REVIEW

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import React from 'react';
+
+export default class Display extends React.Component {
+  render() {
+    return (
+      <div style={{ border: 'dotted thick blue' }}>
+        {this.props.value}
+      </div>
+    );
+  }
```
Comment (SunnyAsar) [inline src/components/Display.js:12]: lets use propTypes to make sure the value coming in as props is a string and let's use defaultProps to set a default value of '0'

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import React from 'react';
+
+export default class Button extends React.Component {
+  render() {
+    return (
+      <div style={{ border: 'dotted thin red', display: 'inline-block' }}>
+        {this.props.name}
+      </div>
+    );
+  }
```
Comment (SunnyAsar) [inline src/components/Button.js:12]: lets set propType here as well for the name

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hello @oscarnava  👋 
Good job on the milestone.
- For the finish please look at the comments below and update accordingly.
- Also remember to use the description of the PR to add a summary of what has been done in the milestone.

**Please request another review when the updates are in place.**
Happy Coding.

File: src/components/Display.js (hunk 1 above)
Comment (oscarnava) [reply src/components/Display.js:12]: I tried using them, but the linter complained about them being deprecated, that's why I omitted them. :-)

Comment (oscarnava) [reply src/components/Display.js:12]: Oh! Now I get it: I was using React.PropTypes as explained in one tutorial; that's why it was appearing as deprecated!

Comment (SunnyAsar) [reply src/components/Display.js:12]: Nice catch 👍

**Review by SunnyAsar — APPROVED** [review]
# Hi @oscarnava 👋 

### The update looks good. Please proceed 👍 
Happy Coding.

**Pull request merged**